docs/metrics.md

```diff
@@ -1,5 +1,8 @@
 # SSP Operator metrics
 
+### cnv:vmi_status_running:count
```

Comment:
is the name correct?

Reply:
yes, it does follow the conventions we are using now, but its a old metric, so we would need a deprecation system

Reply:
yes, its a old name, to change it we would need a deprecation policy